src/pages/Contributing/Install/index.md

```diff
@@ -20,8 +20,8 @@ please folow the this [guide](https://nodejs.org/en/download/package-manager/)
 
 ### Note about the project architecture
 
-The project Use a monorepo architecture using [lerna](https://github.com/lerna/lerna/).  
-Lerna help us to manage the distribution and the versionning of multiple packages into the same repository.  
```

Comment:
(everywhere) don't remove two spaces they are line breaks in markdown

Reply:
Fixed